solutions/standard/main.tf

```diff
@@ -103,3 +103,62 @@ module "redis" {
   service_credential_names      = var.service_credential_names
   backup_crn                    = var.backup_crn
 }
+
+# create a service authorization between Secrets Manager and the target service (Databases for Redis)
+resource "ibm_iam_authorization_policy" "secrets_manager_key_manager" {
+  count                       = var.skip_rd_sm_auth_policy || var.existing_secrets_manager_instance_crn == null ? 0 : 1
+  source_service_name         = "secrets-manager"
+  source_resource_instance_id = local.existing_secrets_manager_instance_guid
+  target_service_name         = "databases-for-redis"
+  target_resource_instance_id = module.redis.guid
+  roles                       = ["Key Manager"]
+  description                 = "Allow Secrets Manager to manage key for the databases-for-redis instance"
```

Comment:
The description in elasticsearch is better since it makes it clear this is an instance to instance policy.